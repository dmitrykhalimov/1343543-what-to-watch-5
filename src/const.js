export const ratingRanks = [
  {
    'Bad': 2.9
  },
  {
    'Normal': 4.9
  },
  {
    'Good': 7.9
  },
  {
    'Very good': 9.9
  },
  {
    'Awesome': 10
  }
];
