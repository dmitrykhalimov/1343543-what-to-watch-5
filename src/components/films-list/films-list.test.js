import React from "react";
import renderer from "react-test-renderer";
import FilmsList from "./films-list";
import {TEST_MOCKS} from "../../const";
import {MemoryRouter} from 'react-router-dom';

describe(`FilmsList`, () => {
  it(`Should FilmsList render correctly`, () => {
    const tree = renderer
      .create(
          <MemoryRouter>
            <FilmsList
              films={TEST_MOCKS.films}
              maxQuantity={TEST_MOCKS.maxQuantity}
            />
          </MemoryRouter>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
