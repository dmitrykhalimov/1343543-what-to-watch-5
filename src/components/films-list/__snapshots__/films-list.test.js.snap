// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmsList Should FilmsList render correctly 1`] = `
<div
  className="catalog__movies-list"
>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      className="small-movie-card__link"
      href="/films/29"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        <video
          autoPlay={false}
          height="175"
          muted={true}
          poster="https://thenewswheel.com/wp-content/uploads/2014/08/Interstate-60-review-BMW-convertible-James-Marsden-8-ball.jpg"
          src="https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm"
          width="280"
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <span
          className="small-movie-card__link"
        >
          Interstate 60: Episodes of the Road
        </span>
      </h3>
    </a>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      className="small-movie-card__link"
      href="/films/29"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        <video
          autoPlay={false}
          height="175"
          muted={true}
          poster="https://thenewswheel.com/wp-content/uploads/2014/08/Interstate-60-review-BMW-convertible-James-Marsden-8-ball.jpg"
          src="https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm"
          width="280"
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <span
          className="small-movie-card__link"
        >
          Interstate 60: Episodes of the Road
        </span>
      </h3>
    </a>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      className="small-movie-card__link"
      href="/films/29"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        <video
          autoPlay={false}
          height="175"
          muted={true}
          poster="https://thenewswheel.com/wp-content/uploads/2014/08/Interstate-60-review-BMW-convertible-James-Marsden-8-ball.jpg"
          src="https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm"
          width="280"
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <span
          className="small-movie-card__link"
        >
          Interstate 60: Episodes of the Road
        </span>
      </h3>
    </a>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      className="small-movie-card__link"
      href="/films/29"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        <video
          autoPlay={false}
          height="175"
          muted={true}
          poster="https://thenewswheel.com/wp-content/uploads/2014/08/Interstate-60-review-BMW-convertible-James-Marsden-8-ball.jpg"
          src="https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm"
          width="280"
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <span
          className="small-movie-card__link"
        >
          Interstate 60: Episodes of the Road
        </span>
      </h3>
    </a>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
  >
    <a
      className="small-movie-card__link"
      href="/films/29"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        <video
          autoPlay={false}
          height="175"
          muted={true}
          poster="https://thenewswheel.com/wp-content/uploads/2014/08/Interstate-60-review-BMW-convertible-James-Marsden-8-ball.jpg"
          src="https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm"
          width="280"
        />
      </div>
      <h3
        className="small-movie-card__title"
      >
        <span
          className="small-movie-card__link"
        >
          Interstate 60: Episodes of the Road
        </span>
      </h3>
    </a>
  </article>
</div>
`;
